wp-content/plugins/wporg-learn/inc/export.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/**
+ * Allow some raw data to be exposed in the REST API for certain post types, so that developers can import
+ * a copy of production data for local testing.
+ */
+
+namespace WPOrg_Learn\Export;
+
+defined( 'WPINC' ) || die();
+
+add_action( 'rest_api_init', function() {
+	// Important: only expose raw post content for specific post types
+	register_raw_content_for_post_type( 'lesson-plan' );
+	register_raw_content_for_post_type( 'wporg_workshop' );
+} );
+
+
+
+function register_raw_content_for_post_type( $post_type ) {
+
+	register_rest_field(
+		$post_type,
+		'content_raw',
+		[
+			'get_callback' => __NAMESPACE__.'\show_post_content_raw',
+			'schema'       => [
+				'type' => 'string',
+				'context' => [ 'export' ]
```

Comment:
```suggestion
				'context' => [ 'wporg_export' ]
```

This might be a bit more future-proof, in case Core adds an `export` context in the future.